reliability-engineering/terraform/modules/concourse-secrets-admin/variables.tf

```diff
@@ -0,0 +1,25 @@
+data "aws_caller_identity" "account" {}
+
+variable "deployment" {
+  type = string
+}
+
+variable "name" {
```

Comment:
Suggest calling this `concourse_team_name`

Reply:
Accepted and changed.